docs/modules/ROOT/pages/how-tos/update_maintenance/automated-upgrades-at-vshn.adoc

```diff
@@ -0,0 +1,25 @@
+= Automated Upgrades at VSHN
+
+[IMPORTANT]
+----
+This page is VSHN specific and uses VSHN configuration.
+
+To implement your own automated upgrades, please refer to the underlying https://hub.syn.tools/openshift-upgrade-controller/references/parameters.html#_upgrade_configs[components documentation].
+----
```

Comment:
Only thing: maybe use the infobox syntax
https://kb.vshn.ch/oc4/how-tos/update_maintenance.html